go/vt/discovery/healthcheck.go

```diff
@@ -149,6 +149,7 @@ func ParseTabletURLTemplateFromFlag() {
 func init() {
 	servenv.OnParseFor("vtgate", registerDiscoveryFlags)
 	servenv.OnParseFor("vtcombo", registerDiscoveryFlags)
+	servenv.OnParseFor("vtctld", registerDiscoveryFlags)
```

Comment:
do we need `vtctl` as well, or _just_ `vtctld` (i assume just `vtctld`, but want to make sure we've thought about it)

Reply:
I don't think `vtctl` requires it, but there is something in `go/vt/throttler/demo/throttler_demo.go` which is also using the healthcheck. I don't if we need to add it 🤷‍♂️

Reply:
`Vttablet` too